packages/traceability-schemas/schemas/UsdaSc6.json

```diff
@@ -50,13 +50,13 @@
             "type": "string"
         },
         "dateOfEntry": {
-            "$comment": "{\"term\": \"dateOfEntry\", \"@id\": \"https://w3id.org/traceability#dateOfEntry\", \"@type\": \"https://w3id.org/traceability#DateTime\"}",
+            "$comment": "{\"term\": \"dateOfEntry\", \"@id\": \"https://w3id.org/traceability#dateOfEntry\", \"@type\": \"http://www.w3.org/2001/XMLSchema#dateTime\"}",
             "title": "dateOfEntry",
             "description": "Date when the the package entered the port of entry.",
             "type": "string"
         },
         "signatureDate": {
-            "$comment": "{\"term\": \"signatureDate\", \"@id\": \"https://w3id.org/traceability#signatureDate\", \"@type\": \"https://w3id.org/traceability#DateTime\"}",
+            "$comment": "{\"term\": \"signatureDate\", \"@id\": \"https://w3id.org/traceability#signatureDate\", \"@type\": \"http://www.w3.org/2001/XMLSchema#dateTime\"}",
```

Comment:
I question why we have this here... and why its not the `issuanceDate` or `proof.created` values... this is why Credential Types are needed not just JSON-LD Class types... still its orthogonal to the purpose of this PR.

Reply:
My understanding is this is a 1-1 from a physical document, I could be wrong though